test/functional/mempool_accept.py

```diff
@@ -98,7 +99,7 @@ def run_test(self):
         tx.deserialize(BytesIO(hex_str_to_bytes(raw_tx_final)))
         self.check_mempool_result(
             result_expected=[{'txid': tx.rehash(), 'allowed': True}],
-            rawtxs=[bytes_to_hex_str(tx.serialize())],
+            rawtxs=[tx.serialize().hex()],
```

Comment:
14964: Why's this being changed? I remember this being a thing in a somewhat recent PR, was this an issue that was introduced when you rebased?

Reply:
bitcoin#14954 was already backported and it contained a scripted diff that would have made that change. The bytes_to_hex_str function was removed so it caused a test to fail, which is how I found it.
bitcoin: fa6bf21f5e ("scripted-diff: test: Use py3.5 bytes::hex() method", 2019-02-18)

Reply:
Correct, however, this change should be in 14926 instead of 14964

Reply:
Sorry, squashed in the wrong order. Would you prefer I fix this, change the message of 14573, and remove 14608 in a seperate PR or force push it to this one?

Reply:
And should I squash them at that time, or after you have another look at it

Reply:
You can do it all in one go.